airbyte-cdk/python/cdk-migrations.md

```diff
@@ -1,5 +1,94 @@
 # CDK Migration Guide
 
+## Upgrading to 6.x.x
```

Comment:
Can we validate with @bnchrch what is the impact on the manifest-only sources and what is the migration path for them?

Reply:
yep, i'll add a note here: But it sounds like `source-declarative-manifest` is pinned to an explicit, version and as for connector builder, that one actually doesn't use `YamlDeclarativeSource` it uses `SourceDeclarativeManifest` which doesn't contain breaking changes. I think it's up for debate whether that should switch over, but for now it would be non breaking for there